dashboard/src/util/Loading.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Dimmer, Loader, Segment } from 'semantic-ui-react';
+
+const Loading = ({message}) => (
+        <Segment>
+        <Dimmer inverted active>
+          <Loader inverted>{message || "Loading..."}</Loader>
+        </Dimmer>
+        <div style={{minHeight:"3em"}}/>
+      </Segment>
+      );
```

Comment:
BTW the indenting is a bit messy. Have you heard of our lord and saviour, Prettier? https://prettier.io/